framework/src/play/jobs/Job.java

```diff
@@ -164,7 +164,19 @@ public void every(String delay) {
      *            time in seconds
      */
     public void every(int seconds) {
-        JobsPlugin.executor.scheduleWithFixedDelay(this, seconds, seconds, TimeUnit.SECONDS);
+        // Wrap this job in a Runnable that catches and ignores all thrown exception.
+        // If there's an unhandled exception, then the executor would suppress all
+        // further executions, violating the contract.  Note that this does
+        // not interfere with the normal handling of onException().
+        Runnable neverThrowingRunnable = new Runnable() {
```

Comment:
first try to avoid anonymous class wrapper creation if you can just use a flag: https://gist.github.com/flybyray/a7d61c66cd666c8aa391b3a0fb92425a#file-job-java-L168
https://gist.github.com/flybyray/a7d61c66cd666c8aa391b3a0fb92425a#file-job-java-L182
=> this will still log the executionerror, which should be logged !!!